strutil/strutil.go

```diff
@@ -22,13 +22,35 @@ package strutil
 import (
 	"fmt"
 	"io"
+	"math/rand"
 	"sort"
 	"strconv"
 	"strings"
+	"time"
 	"unicode"
 	"unicode/utf8"
 )
 
+func init() {
+	// golang does not init Seed() itself
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
+const letters = "BCDFGHJKLMNPQRSTVWXYbcdfghjklmnpqrstvwxy0123456789"
+
+// MakeRandomString returns a random string of length length
+//
+// The vowels are omitted to avoid that words are created by pure
```

Comment:
I was staring at the 'letters' in order to understand why some are missing, and only found the explanation here. Would it perhaps be better to move this up to the const declaration?